scripts/imageSize.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+
+let fileArray = [];
```

Comment:
1) Instead of storing it to an external array, I think we can return the result directly.  So, we could remove this.  The thought process follows to the next comment.